src/main/java/Duke.java

```diff
@@ -1,10 +1,59 @@
+import java.util.Scanner;  // Import the Scanner class
 public class Duke {
+
+    public static void drawLine() {  // Draw horizontal lines
+        for (int i = 0; i < 30; i++) {
+            System.out.print("_");
+        }
+        System.out.println();
+    }
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        drawLine();
+
+        System.out.println("Hello! I'm Duke");  // Greetings
+        System.out.println("What can I do for you?");
```

Comment:
Avoid the usage of magic literals, consider using a constant to store the string literals instead. [See here](https://nus-cs2113-ay2223s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-magic-numbers)